sklearn/metrics/cluster/supervised.py

```diff
@@ -52,10 +56,8 @@ def check_clusterings(labels_true, labels_pred):
     if labels_pred.ndim != 1:
         raise ValueError(
             "labels_pred must be 1D: shape is %r" % (labels_pred.shape,))
-    if labels_true.shape != labels_pred.shape:
-        raise ValueError(
-            "labels_true and labels_pred must have same size, got %d and %d"
-            % (labels_true.shape[0], labels_pred.shape[0]))
+    check_consistent_length(labels_true, labels_pred)
```

Comment:
That removes the check about the potential second dimension though? Not sure if it matters...